docs/vector.rst

```diff
@@ -13,25 +13,25 @@ In fact, ``Vector`` is simply an alias of ``Matrix``, just with ``m = 1``. See :
 Declaration
 -----------
 
-As global fields of vectors
+As global vector fields
 +++++++++++++++++++++++++++
 
 .. function:: ti.Vector.field(n, dtype, shape = None, offset = None)
 
     :parameter n: (scalar) the number of components in the vector
     :parameter dtype: (DataType) data type of the components
-    :parameter shape: (optional, scalar or tuple) shape the field of vectors, see :ref:`tensor`
+    :parameter shape: (optional, scalar or tuple) shape of the vector field, see :ref:`tensor`
     :parameter offset: (optional, scalar or tuple) see :ref:`offset`
 
-    For example, this creates a 5x4 field of 3 component vectors:
+    For example, this creates a ``5x4`` vector field which has a 3 component vector on each entry:
```

Comment:
```suggestion
    For example, this creates a ``5x4`` vector field composed of 3 component vector:
```
Concern: will the term 3D fields overlaps 3D vectors?
How about: 3D fields and 3 component vectors?
Thoughts? @yuanming-hu @Rullec

Reply:
Thanks for this reminder. Would you please explain what are you referring to by "3D fields and 3 component vectors"?
Basically It's not very easy to describate the shape of a "vector field", because we usually both need to specify the shape of the field part and the shape of its vector.

Reply:
@Rullec, here is an example that may help you understand what @archibate  referred to as '3d field':
Take the  3d lid-driven flow as an easy example. The computational domain is a discrete 3d space, and on each node, we can have a 3d-vector (ux,uy,uz) for the velocity and a scalar for pressure P. Let's assume the shape is '10x10x10'. In this case, we have a 3d-vector field of the shape of '10x10x10' and a scalar pressure field(of the same shape).

@archibate, I don't see any confusion about the overlapping '3d prefix'. But if you are concerned about describing precisely, I would suggest we **never describe the dimension of the field shape, instead, add 'of the shape of axbxc' as a postfix**, because **the field dimension does not always correspond to the space dimension** and can become very misleading.

Commonly seen in FEM, you import an external mesh with a very complicated 3d shape. It's preferable to merely create 1d fields with the same length as the number of nodes to store values. The shape of the field no longer represents the dimension of the space.

Reply:
Thanks for your reply! It is a very outstanding example that makes me understand it at once. :)
I would leave this problem here, until we can come to a conclusion!

Reply:
I would suggest here, but it's up to you to decide :>

```suggestion
    For example, this creates a 3-d vector field of the shape of ``5x4``:
```